flytekit/configuration/sdk.py

```diff
@@ -20,7 +23,11 @@
 TODO: Explain how this would be used to extend the SDK
 """
 
-LOCAL_SANDBOX = _config_common.FlyteStringConfigurationEntry("sdk", "local_sandbox", default="/tmp/flyte")
+LOCAL_SANDBOX = _config_common.FlyteStringConfigurationEntry(
```

Comment:
can we not use TempDir - https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory